packages/react-native-lockdown/src/index.js

```diff
@@ -0,0 +1,200 @@
+'use strict'
+
+/**
+ * React Native lockdown entry point.
+ *
+ * @packageDocumentation
+ */
+
+/**
+ * @typedef {import('@react-native/metro-config').SerializerConfig} SerializerConfig
+ */
```

Comment:
```suggestion
/**
 * @import {SerializerConfig} from '@react-native/metro-config'
 */
```

This is the new-ish better way to do this sort of thing